src/js/themes/base.js

```diff
@@ -517,17 +520,34 @@ export const generate = (baseSpacing = 24, scale = 6) => { // 24
       xlarge: { ...fontSizing(2) },
     },
     radioButton: {
-      check: {
-        // color: { dark: undefined, light: undefined },
-      },
       border: {
         color: {
           dark: 'rgba(255, 255, 255, 0.5)',
           light: 'rgba(0, 0, 0, 0.15)',
         },
-        radius: '100%',
         width: '2px',
       },
+      check: {
+        radius: '100%',
+        // color: { dark: undefined, light: undefined },
+        // extend: undefined,
+      },
+      hover: {
+        border: {
+          color: {
+            dark: css`${props => normalizeColor('white', props.theme)}`,
```

Comment:
I think this means we will normalize twice. I think you can just put `dark: 'white'` here.

Reply:
👍